ner_v1/detectors/numeral/number/number_detection.py

```diff
@@ -118,14 +144,18 @@ def _detect_number_format(self):
                 input: "I want to purchase 30 units of mobile and 40 units of Television"
                 output: (['30', '40'], ['30','40'])
 
-        Note: Currently,we can detect numbers from 1 digit to 3 digit
+        Note: By default we detect numbers from 1 digit to 3 digit
         """
         number_list = []
         original_list = []
-        patterns = re.findall(r'\s([0-9]{1,3})[\s|,]', self.processed_text.lower())
+        patterns = re.findall(r'\s([0-9]{' + str(self.min_digit) + ',' + str(self.max_digit) + '})[\s|,]',
```

Comment:
Minor:
r'\s([0-9]{{{0},{1}}})[\s|,]'.format(self.min_digit, self.max_digit) looks much cleaner

Also, the `\s` can be switched out with `\b`